client/lobby/OptionsTab.cpp

```diff
@@ -493,7 +497,9 @@ OptionsTab::PlayerOptionsEntry::PlayerOptionsEntry(const PlayerSettings & S)
 
 	if(SEL->screenType != ESelectionScreen::scenarioInfo && SEL->getPlayerInfo(s.color.getNum()).canHumanPlay)
 	{
-		flag = std::make_shared<CButton>(Point(-43, 2), flags[s.color.getNum()], CGI->generaltexth->zelp[180], std::bind(&IServerAPI::setPlayer, CSH, s.color));
+		flag = std::make_shared<CButton>(Point(-43, 2), flags[s.color.getNum()], CGI->generaltexth->zelp[180],
+			std::bind(& OptionsTab::onSetPlayerClicked, & parentTab, s));
```

Comment:
As far as I know we do not need space after & when it is used to take address (unary operation, but we need spaces in declarations)